src/Tizen.Uix.Sticker/Tizen.Uix.Sticker/GroupImage.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+* Copyright (c) 2022 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+using System;
+
+namespace Tizen.Uix.Sticker
+{
+    /// <summary>
+    /// This structure represents a image of sticker group.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public struct GroupImage
+    {
+        /// <summary>
+        /// The public constructor.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        /// <param name="groupName">The group name for setting group image.</param>
+        /// <param name="uriType">The URI type of the image file.</param>
+        /// <param name="uri">The URI of the image file.</param>
+        public GroupImage(string groupName, UriType uriType, string uri)
+        {
+            GroupName = groupName;
+            UriType = uriType;
+            Uri = uri;
+        }
+
+        /// <summary>
+        /// The group name for setting group image.
```

Comment:
"setting"? maybe just "The name of the group".

Reply:
fixed.